src/main/java/persistence/sql/dml/Insert.java

```diff
@@ -0,0 +1,96 @@
+package persistence.sql.dml;
+
+import persistence.sql.dialect.Dialect;
+import persistence.sql.mapping.Column;
+import persistence.sql.mapping.Table;
+import persistence.sql.mapping.Value;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Insert {
+
+    private final Table table;
+
+    private final List<Column> columns;
+
+    private final List<Column> pkColumns;
+
+    public Insert(final Table table) {
+        this.table = table;
+        final List<Column> tempColumns = table.getColumns();
+
+        this.columns = tempColumns.stream().filter(Column::isNotPk).collect(Collectors.toList());
+        this.pkColumns = tempColumns.stream().filter(Column::isPk).collect(Collectors.toList());
+    }
+
+    public Table getTable() {
+        return this.table;
+    }
+
+    public List<Column> getColumns() {
+        return this.columns;
+    }
+
+    public List<Column> getPkColumns() {
+        return this.pkColumns;
+    }
+
+    public String columnNameClause(final Dialect dialect) {
+        final String columnNameClause = columns.stream()
+                .map(Column::getName)
+                .collect(Collectors.joining(", "));
+
+        final StringBuilder clause = new StringBuilder()
+                .append(columnNameClause);
+
+        if (clause.length() > 0) {
+            clause.append(", ");
+        }
```

Comment:
```suggestion
        if (!classe.isEmpty()) {
            clause.append(", ");
        }
```
이런 사소한 부분이 아니면 잘 해주셔서 크게 피드백해야 할 부분이 보이지 않네요 👍